CONTRIBUTING.rst

```diff
@@ -13,42 +13,28 @@ TL;DR
 
 .. code-block:: console
 
-    $ # Export libpq env vars for PG connection
-    $ export PGDATABASE=procrastinate PGHOST=localhost PGUSER=postgres
-
-    $ # Launch PostgreSQL within Docker
+    $ # Launch services within Docker
     $ docker-compose up -d
 
-    $ # Explore tox entrypoints
-    $ tox -l
+    $ # Create procrastinate databases
+    $ docker-compose run --rm procrastinate migrate
+
+    $ # Launch demo
+    $ docker-compose run --rm procrastinate -h
 
-    $ # You can do things without tox too:
+    $ # Launch tests from the docker container
+    $ docker-compose exec procrastinate pytest
 
+    $ # Outside of docker containers:
```

Comment:
I find this a bit confusing. It sounds like the "outside of docker containers" commands are the next commands to run, while they only make sense if Docker is not used.